package.json

```diff
@@ -13,6 +13,7 @@
     "path": "node_modules/cz-conventional-changelog/"
   },
   "dependencies": {
+    "@bahmutov/parse-github-repo-url": "^0.1.0",
```

Comment:
Perhaps a silly question, but why another parse library?

Just searching Google, there are quite a few out there.

Also, if it's not github specific (your README mentions GItLab), why not remove the `github` reference from the package name?